ld: framework not found Mpv

When run with target macos, throws this error

```
Launching lib/main.dart on macOS in debug mode...
Running pod install...
Building macOS application...
Command CompileSwiftSources failed with a nonzero exit code
Command CompileSwiftSources failed with a nonzero exit code
ld: warning: directory not found for option '-F/Users/eyem/Downloads/shortcam/shortcam_lite_frontend/macos/Pods/../Flutter/ephemeral/.symlinks/plugins/media_kit_libs_macos_video/macos/Frameworks/.symlinks/mpv/macos'
ld: framework not found Mpv
clang: error: linker command failed with exit code 1 (use -v to see invocation)
** BUILD FAILED **

Error: Build process failed
```

Further investigate, found out that theres error in here:
https://github.com/media-kit/media-kit/blob/63c6ebe8366db7ecfbd13ab9ce76b11dd86dae48/libs/macos/media_kit_libs_macos_video/macos/create_framework_symlinks.sh#L64

`NAME="$(echo "${SLUG}" | cut -d '-' -f 1 -f 3)"`
When i try to run that in terminal it throws error
```
cut: only one list may be specified
Try 'cut --help' for more information.
```

The solution is obvious the `-f 1 -f 3` should be `-f 1,3`:
`NAME="$(echo "${SLUG}" | cut -d '-' -f 1,3)"`

After that run it:
`sh create_framework_symlinks.sh Frameworks/Mpv.xcframework Frameworks/.symlinks/mpv`
Then its working.

Somehow i must manually run above script once time.
But if i clear the flutter cache (package cache), i need to manually run above script, to symlink the mpv.
Somehow i could not just click run button inside android studio without error, the first time i use media kit.
But after successfull running, its cached, no need to run it anymore.

--- a/libs/ios/media_kit_libs_ios_audio/ios/create_framework_symlinks.sh
+++ b/libs/ios/media_kit_libs_ios_audio/ios/create_framework_symlinks.sh
@@ -61,7 +61,7 @@ create_framework_symlinks() {
 
     find "${SRC_DIR}" -mindepth 1 -maxdepth 1 -type d | while read SRC; do
         SLUG="$(basename "${SRC}")"
-        NAME="$(echo "${SLUG}" | cut -d '-' -f 1 -f 3)"
+        NAME="$(echo "${SLUG}" | cut -d '-' -f 1,3)"
 
         SRC_RELATIVE="$(relpath "${SYMLINKS_DIR}" "${SRC}")"
 

--- a/libs/ios/media_kit_libs_ios_video/ios/create_framework_symlinks.sh
+++ b/libs/ios/media_kit_libs_ios_video/ios/create_framework_symlinks.sh
@@ -61,7 +61,7 @@ create_framework_symlinks() {
 
     find "${SRC_DIR}" -mindepth 1 -maxdepth 1 -type d | while read SRC; do
         SLUG="$(basename "${SRC}")"
-        NAME="$(echo "${SLUG}" | cut -d '-' -f 1 -f 3)"
+        NAME="$(echo "${SLUG}" | cut -d '-' -f 1,3)"
 
         SRC_RELATIVE="$(relpath "${SYMLINKS_DIR}" "${SRC}")"
 

--- a/libs/macos/media_kit_libs_macos_audio/macos/create_framework_symlinks.sh
+++ b/libs/macos/media_kit_libs_macos_audio/macos/create_framework_symlinks.sh
@@ -61,7 +61,7 @@ create_framework_symlinks() {
 
     find "${SRC_DIR}" -mindepth 1 -maxdepth 1 -type d | while read SRC; do
         SLUG="$(basename "${SRC}")"
-        NAME="$(echo "${SLUG}" | cut -d '-' -f 1 -f 3)"
+        NAME="$(echo "${SLUG}" | cut -d '-' -f 1,3)"
 
         SRC_RELATIVE="$(relpath "${SYMLINKS_DIR}" "${SRC}")"
 

--- a/libs/macos/media_kit_libs_macos_video/macos/create_framework_symlinks.sh
+++ b/libs/macos/media_kit_libs_macos_video/macos/create_framework_symlinks.sh
@@ -61,7 +61,7 @@ create_framework_symlinks() {
 
     find "${SRC_DIR}" -mindepth 1 -maxdepth 1 -type d | while read SRC; do
         SLUG="$(basename "${SRC}")"
-        NAME="$(echo "${SLUG}" | cut -d '-' -f 1 -f 3)"
+        NAME="$(echo "${SLUG}" | cut -d '-' -f 1,3)"
 
         SRC_RELATIVE="$(relpath "${SYMLINKS_DIR}" "${SRC}")"
 

